src/search.css

```diff
@@ -1,8 +1,19 @@
+.govuk-phase-banner .app-align-items-center {
+  height: 40px;
+}
+
+.govuk-phase-banner .app-search-inline-phase-banner {
+  position: sticky;
+  top: 100%;
+  z-index: 1;
+}
+
 .sirius-search {
-  position: absolute;
+  top: 0;
+  left: 0;
+  position: static;
```

Comment:
By making this static, you're having to navigate around overflow traps which are likely to cause problems later (for example, this only works if the search is being contained by a phase-banner which won't always be the case).

I'd suggest you instead want to make the <form> parent relative, then you can use absolute positioning (which will be relative to that parent) and you won't need to amend phase-banner CSS.